exercicios/exercicio-alunas/AdaCruz/script.js

```diff
@@ -0,0 +1,31 @@
+let nomeusuario = prompt('Digite seu nome');
+
+function vazia(str) {
+    if(str)
+    return false;
+    return true;

```

Comment:
@AdaFPC aqui você tem dois `return` e o browser só vai considerar o último, então acho que você pode remover o `return false;`

Reply:
A function `vazia` só retorna `true` se a parâmetro for `null`, vazia, `undefined`  ou `false`. Caso coloque algum valor ela retorna `false`. Por isso eu coloquei dois `return`.

Reply:
@AdaFPC entendi, você usou uma sintaxe diferente para representar o if/else, correto? Então pode desconsiderar meu comentário.

```js
function vazia(str) {
if(str) {
    return false;
} else {
   return true;
}  
```

Reply:
Foi isso mesmo.